aidial_sdk/exceptions.py

```diff
@@ -6,7 +7,7 @@ def __init__(
         self,
         message: str,
         status_code: int = 500,
-        type: str = "runtime_error",
+        type: Optional[str] = "runtime_error",
```

Comment:
Why is it optional? I don’t remember any OpenAI error without type

Reply:
Because it's [optional](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/stable/2024-06-01/inference.yaml#L569) in Azure OpeAI API spec.

Reply:
But `message` is optional too :) Ok, let's allow it, I don't see issues